bin/check_sorted.py

```diff
@@ -1,12 +1,19 @@
+# -*- coding=utf-8 -*-
 #!/usr/bin/env python
 import fileinput
 import re
+import sys
+import io
 
 """
 Check if each section separated by comment has alphabetically sorted lines.
 """
 prev_line = None
-for line in fileinput.input():
+# Make this script to run in Python 2 and Python 3
+with io.open(sys.argv[1], 'r', encoding='utf-8') as f:
+    filestream = f.readlines()
+
+for line in filestream:
```

Comment:
Is this necessary for Python 2.7? `python bin/check_sorted.py youslist.txt` works with Python 2.7.15 on Ubuntu 18.04.2 LTS and Windows 10.

Reply:
```python
for line in fileinput.input():
```
The previous code doesn't work in Python 3 on Windows 10. I've got the below error messages when testing. 

![error](https://user-images.githubusercontent.com/14074081/58746015-2d353a00-8494-11e9-8e17-91810fb5e826.PNG)

It's because basically Python 3 opens files as 'ASCII' format, I guess.